src/query/service/src/servers/mysql/mysql_federated.rs

```diff
@@ -177,7 +177,7 @@ impl MySQLFederated {
     fn federated_mixed_check(&self, query: &str) -> Option<DataBlock> {
         let rules: Vec<(&str, Option<DataBlock>)> = vec![
             (
-                "(?i)^(SELECT VERSION())",
+                r"(?i)^(SELECT VERSION\(\s*\))",
```

Comment:
Can you give a raw query sent from MySQL?

Reply:
```
SELECT version_name,\n       os,\n       p_date, file_type, video_succ_cnt\nFROM xxx\nWHERE p_date >= '20190101'\n  AND p_date <= '20191010'\nORDER BY version_name,\n         os,\n         p_date, file_type, video_succ_cnt\nLIMIT 100
```

Reply:
The old regex `(?i)^(SELECT VERSION())` is wrong and will match all query that start with `select version`

![image](https://user-images.githubusercontent.com/5351546/195042568-d5c6dfe4-63a2-43a3-9def-a28653ed3d57.png)

Reply:
👍 
Could you add a test for this query in https://github.com/datafuselabs/databend/blob/main/src/query/service/tests/it/servers/mysql/mysql_federated.rs

Reply:
updated